internal/resource/helper.go

```diff
@@ -56,6 +57,17 @@ func parseURL(s *v1.Service) error {
 	return nil
 }
 
+func isReference(v interface{}) bool {
```

Comment:
One you fix [this](https://github.com/Kong/koko/pull/449/files#r983158470), you can also simplify this function.

```
func isReference(v string) bool {
	r := regexp.MustCompile(typedefs.ReferencePattern)
	return r.MatchString(v)
}
```

Reply:
done